english/light-fire/dream/dream.md

```diff
@@ -0,0 +1,15 @@
+You decided to move forward...
+
+That's clever.
+
+But wait. The rat starts growing in size.
+
+You better look scared. 
+
+The dark place has some light now.Not the spotlight!!
+
+A fire ring around your battlefield. Yes,a battlefield!! 
+
+[Start fighting](fight/fight.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.